util.go

```diff
@@ -130,3 +134,22 @@ func durationUntilNextDay() time.Duration {
 	endOfDay = endOfDay.Add(1 * time.Second)
 	return time.Until(endOfDay)
 }
+
+func toJSON(data []byte) ([]byte, error) {
+	if hasJSONPrefix(data) {
+		return data, nil
+	}
+	return yaml.YAMLToJSON(data)
+}
+
+// hasJSONPrefix returns true if the provided buffer appears to start with
+// a JSON open brace.
```

Comment:
Is this better than just relying on the file extension?  Are we trying to support users not specifying a file extension on their config file?

Reply:
Currently we don't rely on the file extension at all. By default we try to read "config.json" from the working directory but you could also specify a different location or filename by using the parameter `-configuration={some_name}`. This is used for the docker image for example.

If we start supporting different file formats we have decide how and where to distinguish these. This is not only relevant for the content structure and the file extension but also for the default naming.

I see the options

1. Rename default to `config` and decide on the format by it's content
2. Use the extension to define the content structure

I tend towards option two because this is why file extensions exist in the first place. Not sure about the default though... Tend to stick to the default JSON for now.